src/components/molecules/MarketStats.tsx

```diff
@@ -62,20 +86,9 @@ export default function MarketStats(): ReactElement {
       Total of <strong>{stats?.datasets.total}</strong> data sets & unique
       datatokens published by <strong>{stats?.owners}</strong> accounts.
       <br />
-      <PriceUnit
-        price={`${stats?.ocean}`}
-        small
-        className={styles.total}
-        conversion
-      />{' '}
-      and{' '}
-      <PriceUnit
-        price={`${stats?.datatoken}`}
-        symbol="datatokens"
-        small
-        className={styles.total}
-      />{' '}
-      in <strong>{stats?.datasets.pools}</strong> data set pools.
+      <Conversion price={`${totalLockedValue}`} hideApproximationSign /> TVL
+      (total value locked) across <strong>{stats?.datasets.pools}</strong> data
```

Comment:
good point here actually, we need to decide if we should get this from `market-stats` endpoint, or from the The Graph query. In this case, since it's about the pools, it should come from The Graph I think.

Definitely would be most effective to get all data within this component from one source only but not sure that's feasible for all numbers, like number of datasets and accounts.

Reply:
well from the original issue i understand that we settled on just `€1,209,894.27 TVL (total value locked) across 156 dataset pools that contain 1,095,651.439 OCEAN and the datatokens for each pool. ` and drop `Total of 278 data sets & unique datatokens published by 172 accounts.` since that doesn't have that much value and we keep it simple. We have all that info from the graph  (or we will have it at least once it is updated) . So basically this pr is blocked until then